tests/trainer/test_dataloaders.py

```diff
@@ -1119,17 +1119,12 @@ def test_dataloaders_load_only_once(tmpdir):
     assert tracker.mock_calls == [call.val_dataloader(), call.train_dataloader()]
 
 
-def test_dataloaders_load_only_once_val_interval(tmpdir):
+def test_dataloaders_load_only_once_no_sanity_check(tmpdir):
     model = BoringModel()
 
     # logger file to get meta
     trainer = Trainer(
-        default_root_dir=tmpdir,
-        limit_train_batches=10,
-        limit_val_batches=10,
-        val_check_interval=0.3,
-        reload_dataloaders_every_n_epochs=1,
-        max_epochs=3,
+        default_root_dir=tmpdir, limit_train_batches=0.3, limit_val_batches=0.3, num_sanity_val_steps=0, max_epochs=3
```

Comment:
Why change the values for `limit_*_batches?`

Also I'd prefer keeping the the previous `num_sanity_val_steps` value

Reply:
This diff is misleading. Check out the [changes for the second commit](https://github.com/PyTorchLightning/pytorch-lightning/pull/11036/commits/123984d9a1ec72aaf34dd6cd58892c054b96b6ac).

I'm removing the `test_dataloaders_load_only_once_val_interval` test because I realized that the test I added in the first commit [`test_dataloaders_load_every_n_epochs_frequent_val`](https://github.com/PyTorchLightning/pytorch-lightning/pull/11036/commits/4c818369208c2bc5be0fca6f7dd86fa7e82229d0#diff-c439da4427c1b40699495f578c0619586bde3992b3a57aeef80fefe2fa940c1aR1266) checks the same case.

Reply:
Can we merge this test with the one below ?